management/models.py

```diff
@@ -24,15 +24,28 @@ def save(self, *args, **kwargs):
 class PermissionsBase(models.Model):
     """Base model for models that require permissions."""
 
-    owner = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+        help_text="Owner of the object.",
+    )
 
     VISIBILITY_LEVELS = [
         ("public", "Public"),
         ("private", "Private"),
     ]
 
     visibility = models.CharField(
-        max_length=8, choices=VISIBILITY_LEVELS, default="private"
+        max_length=8,
+        choices=VISIBILITY_LEVELS,
+        default="private",
+        null=False,
+        blank=False,
+        help_text="Visibility level of the object. WARNING: Changing this setting will "
```

Comment:
What would happen if someone tried to change an object to private after someone else has already referenced it in another object?

Reply:
I wanted to prevent from an object to be made private after it had been referenced by another one, but didn't found a way of doing it. So, as it is now, nothing will happen: the existing relations will stay but it will not be possible to create new ones.